activerecord/test/cases/migration/compatibility_test.rb

```diff
@@ -606,6 +606,21 @@ def migrate(x)
         connection.drop_table :more_testings rescue nil
       end
 
+      def test_change_table_collation_not_unset_7_0
+        if current_adapter?(:Mysql2Adapter, :TrilogyAdapter)
```

Comment:
The change makes sense to me, but I think we should put the `if` outside of the test so that we don't end up running an empty test for non-{mysql2,trilogy} adapters

Reply:
good call! fixed.